csrc/multidevice/communication.cpp

```diff
@@ -210,6 +210,53 @@ std::string Communication::toInlineString(int indent_size) const {
   return toString(indent_size);
 }
 
+std::ostream& operator<<(std::ostream& os, const P2PCommunicationType& type) {
+  switch (type) {
+    case P2PCommunicationType::send:
+      os << "send";
+      break;
+    case P2PCommunicationType::recv:
+      os << "recv";
+      break;
+    default:
+      NVF_THROW("unrecognized P2PCommunicationType: ", type);
+  }
+  return os;
+}
+
+P2PCommunication::P2PCommunication(
+    IrBuilderPasskey passkey,
+    P2PCommunicationType type,
+    TensorView* buffer,
+    Val* peer,
+    Val* tag)
+    : Expr(passkey) {
+  if (tag == nullptr) {
+    tag = passkey.ir_container_->zeroVal(); // NOLINT
```

Comment:
What did lint complain about? I'd specify the type instead of the generic NOLINT.

Reply:
since I removed tags, this is no longer relevant